### PR TITLE
Support disabled demog question

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This adds support for an additional demog question _are you living with a disability?_ and switches on production mode

### CODE REVIEW

File: code_schemes/disabled.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    {
+      "CodeID": "code-E-720cdb66",
+      "CodeType": "Meta",
+      "MetaCode": "escalate",
+      "DisplayText": "meta: ESCALATE",
+      "NumericValue": -100080,
+      "StringValue": "escalate",
+      "VisibleInCoda": true,
+      "Shortcut": "e"
+    },
```
Comment (as2388) [inline code_schemes/disabled.json:31]: Do we know what data we expect to get back yet? 
It seems yes/no are the obvious labels to start with, but can understand you wanting to wait.

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -44,11 +44,13 @@
     {"RapidProKey": "Gender (Text) - oxfam_wash_s01_demog", "PipelineKey": "gender_raw"},
     {"RapidProKey": "Gender (Time) - oxfam_wash_s01_demog", "PipelineKey": "gender_time"},
     {"RapidProKey": "Age (Text) - oxfam_wash_s01_demog", "PipelineKey": "age_raw"},
-    {"RapidProKey": "Age (Time) - oxfam_wash_s01_demog", "PipelineKey": "age_time"}
+    {"RapidProKey": "Age (Time) - oxfam_wash_s01_demog", "PipelineKey": "age_time"},
+    {"RapidProKey": "Disabled (Text) - oxfam_wash_s01_demog", "PipelineKey": "disabled_raw"},
+    {"RapidProKey": "Disabled (Time) - oxfam_wash_s01_demog", "PipelineKey": "disabled_time"}
   ],
-  "ProjectStartDate": "2020-01-01T00:30:00+03:00",
+  "ProjectStartDate": "2020-11-28T00:30:00+03:00",
```
Comment (as2388) [inline configuration/pipeline_config.json]:
```suggestion
  "ProjectStartDate": "2020-11-28T00:00:00+03:00",
```
?

**Review by as2388 — APPROVED** [review]

File: code_schemes/disabled.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/disabled.json:31]: Yup wanted to be sure. We've done the thematic analysis and have updated this with yes/no.

File: configuration/pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json]: yes thanks

**Pull request merged**